aries_cloudagent/revocation/routes.py

```diff
@@ -424,7 +424,8 @@ async def revoke(request: web.BaseRequest):
     body["notify"] = body.get("notify", context.settings.get("revocation.notify"))
     notify = body.get("notify")
     connection_id = body.get("connection_id")
-    notify_version = body.get("notify_version", "v1_0")
+    body["notify_version"] = body.get("notify_version", "v1_0")
+    notify_version = body["notify_version"]
```

Comment:
Question about this change - basically it changes aca-py behaviour to *always* send a revocation notification if the startup option is set (`--notify-revocation`) vs only sending if a `notify_version` is explicitly set in the revocation payload.

I assume that if you are notifying then you always want to notify.

Reply:
I would think that is right.  And presumably if `notify_version` is not set, it should default to something -- e.g. the latest version?

Reply:
> I would think that is right. And presumably if `notify_version` is not set, it should default to something -- e.g. the latest version?

`v1_0` is the default  (it was the default before however the value wasn't getting passed through to the actual notification code so no notification was getting sent)